render-app/src/main/java/org/janelia/alignment/filter/CompositeFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.janelia.alignment.filter;
+
+import ij.process.ImageProcessor;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Holds a collection of {@link Filter}s to be applied in sequence.
+ *
+ * @author Michael Innerberger
+ */
+public class CompositeFilter implements Filter {
+
+    private List<Filter> filters;
+
+    // empty constructor required to create instances from specifications
+    @SuppressWarnings("unused")
+    public CompositeFilter() {
+        this((List<Filter>) null);
+    }
+
+    public CompositeFilter(final Filter... filters) {
+        this(List.of(filters));
+    }
+
+    public CompositeFilter(final List<Filter> filters) {
+        this.filters = filters;
+    }
+
+    @Override
+    public void init(final Map<String, String> params) {
+        filters = new ArrayList<>(params.size());
+        for (final String serializedFilter : params.values()) {
```

Comment:
You're right, the `values()` are probably in random order (depending on the concrete implementation of the `Map`). Thanks for spotting that!
I kinda prefer the one-entry-per-filter approach since it shows up nicely in the web view. I changed the key to `"filter0": "...", "filter1": "...", ...` and changed `init(...)` so that it retrieves the filters in order. Does this address your concern?